src/components/ui/ShareMenu.jsx

```diff
@@ -13,126 +13,114 @@ const twitterShareUrl = location => {
 
 const menu_height = 3 * 32;
```

Comment:
Do you remember what's the reasoning behind this value ?

Reply:
This seems to be the height of the share menu itself, made of 3 lines (copy, fb, twitter) of approx 32 of height. I guess we put it as a constant because we need it to position the menu before rendering it, but it's not beautiful :D